spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -148,39 +170,27 @@ def test_get_var_properties(kernel):
     assert "'array_ndim': None" in var_properties
 
 
-def test_send_spyder_msg(kernel):
-    """
-    Test publishing custom messages to the Spyder frontend.
-    """
-    spyder_msg_type, content, data = 'TEST', None, None
-    kernel.send_spyder_msg(spyder_msg_type, content, data)
-    log_text = get_log_text(kernel)
-    assert "{'spyder_msg_type': 'TEST'}" in log_text
-
-
 def test_get_value(kernel):
     """Test getting the value of a variable."""
     name = 'a'
-    execute = kernel.do_execute("a = 1", True)
+    kernel.do_execute("a = 124", True)
 
     # Check data type send
-    kernel.get_value(name)
-    log_text = get_log_text(kernel)
-    assert "{'spyder_msg_type': 'data'}" in log_text
+    assert kernel.get_value(name) == 124
 
 
 def test_set_value(kernel):
     """Test setting the value of a variable."""
     name = 'a'
     execute = kernel.do_execute('a = 0', True)
-
+    PICKLE_PROTOCOL = 2
```

Comment:
PICKLE_PROTOCOL should be a constant that we can import from somewhere, as before.